tests/SqlsrvInfoTest.php

```diff
@@ -68,8 +70,7 @@ public function provideFetchTableNames()
                     'spt_fallback_db',
                     'spt_fallback_dev',
                     'spt_fallback_usg',
-                    'spt_monitor',
-                    'spt_values',
```

Comment:
Not sure, but this table is not in the spec, nor was it coming back for me during this test. Presumably this was required for a green run at some point, but this doesn't appear to be the case.